### PR TITLE
Prefer real over nuked releases

### PR DESCRIPTION
**Pull request by duramato**
As talked on irc

### CODE REVIEW

**Comment by SickRage2** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your PR has all of the below criteria checked
- [ ] Give a description on what the PR is for.
- [x] Make sure your PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review

Please make sure you also read [contribution guide](https://github.com/SickRage/SickRage/blob/master/contributing.md) and followed all of the steps.

Thanks!

**Comment by miigotu** [comment]
What about `the real housewives`. Regardless a `REAL` doesnt nuke the original release.

**Comment by duramato** [comment]
@miigotu Thats a good point as that also applies for the repack and propers (dont know with repack but there are shows named proper)
But that shouldnt matter as all results will have the real housewives so prefering one over another wouldnt matter , as there should be only 1 scene release for a specific quality

**Comment by duramato** [comment]
@miigotu problem with proper in the name https://github.com/SickRage/sickrage-issues/issues/274
Fixing this would fix the real too , i think :P

File: sickbeard/search.py [hunk 1]
```diff
@@ -251,8 +251,8 @@ def pickBestResult(results, show):
         elif cur_result.quality in anyQualities and bestResult.quality not in bestQualities and bestResult.quality < cur_result.quality:
             bestResult = cur_result
         elif bestResult.quality == cur_result.quality:
-            if "proper" in cur_result.name.lower() or "repack" in cur_result.name.lower():
-                logger.log(u"Preferring " + cur_result.name + " (repack/proper over nuked)")
+            if "proper" in cur_result.name.lower() or "real" in cur_result.name.lower() or "repack" in cur_result.name.lower():
```
Comment (miigotu) [inline sickbeard/search.py:254]: Arent REAL always uppercase? If so this would be better as `"REAL" in cur_result.name`

Comment (duramato) [inline sickbeard/search.py:254]: Yes think , or at least they should unless the indexer/tracker changes it
Should i change it like you suggested?
That would work too for PROPER and REPACKs

Comment (miigotu) [inline sickbeard/search.py:254]: idc either way, not that important imho

**Pull request merged**